kit/score/results_test.go

```diff
@@ -380,25 +380,25 @@ var scoreGrades = []struct {
 	{
 		in: []*score.Score{
 			{TestName: "A", Score: 10, MaxScore: 10, Weight: 1},
-			{TestName: "B", Score: 05, MaxScore: 05, Weight: 1},
+			{TestName: "B", Score: 0o5, MaxScore: 0o5, Weight: 1},
 			{TestName: "C", Score: 15, MaxScore: 15, Weight: 1},
 		},
 		out:       100,
 		wantGrade: "A",
 	},
 	{
 		in: []*score.Score{
-			{TestName: "A", Score: 05, MaxScore: 10, Weight: 1},
-			{TestName: "B", Score: 05, MaxScore: 05, Weight: 1},
+			{TestName: "A", Score: 0o5, MaxScore: 10, Weight: 1},
```

Comment:
I don't understand these `o` additions here? Is this some formatter thing that automatically adds these? 

The reason I added the `0` prefix was to avoid that positioning gets misaligned. That is, if we want to edit or add more test lines, it is easy to change many lines at once using multi-line editing.

Reply:
I definetly did not add these.
I recall seeing them when I first opened this file, and therefore thought they were supposed to be there.